docs/template-windows.yaml

```diff
@@ -0,0 +1,71 @@
+apiVersion: v1
+kind: Template
+metadata:
+  name: windows-vm-template
+  annotations:
+    description: "OCP kubevirt windows VM template"
+    tags: "kubevirt,ocp,template,windows"
+    miq.github.io/kubevirt-is-vm-template: true
+objects:
+- apiVersion: kubevirt.io/v1alpha1
+  kind: OfflineVirtualMachine
+  metadata:
+    name: ${NAME}
+  spec:
+    template:
+      spec:
+        domain:
+          features:
+            acpi: {}
+            apic: {}
+            hyperv:
+              relaxed: {}
+              vapic: {}
+              spinlocks:
+                spinlocks: 8191
+          clock:
+            utc: {}
+            timer:
+              hpet:
+                present: false
+              pit:
+                tickPolicy: delay
+              rtc:
+                tickPolicy: catchup
+              hyperv: {}
+          firmware:
+            uid: ${UID}
```

Comment:
UID does not seem to be reflected in the parameters. 

Just for completeness: If the uid is not specified, OfflineVirtualMachine can/will try to provide an UID based on the VM name. That has its limits (e.g. vms in differnet namespaces with the same name can get the same UID, which in theory could produce licensing  issues) but it is something which we might for instance use in a future VirtualMachineStatefulSet.

Reply:
+1

Let's dro the uuid from this teomplate for now, we'll sovle it in the scope of OfflineVM

Reply:
Done